modules/ocf_backups/files/upload-to-box

```diff
@@ -353,7 +353,9 @@ def main(args):
     # Clear the screen and start printing the progress bar every second
     if not args.quiet:
         subprocess.call(('clear',))
-        print_progress(multi, stats, 0)
+        progress_thread = threading.Timer(1, print_progress, (multi, stats))
```

Comment:
This is the line I meant, I think this should just be a regular thread, not a timer now.